CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 ## Unreleased
 
+## v0.14.0
+
+### IMPROVEMENTS:
+
+* update dependencies [GH-113](https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/113)
+  * `github.com/go-ldap/ldap/v3` v3.4.6 -> v3.4.8
+  * `github.com/hashicorp/go-hclog` v1.6.2 -> v1.6.3
+  * `github.com/hashicorp/go-secure-stdlib/parseutil` v0.1.7 -> v0.1.8
+  * `github.com/hashicorp/vault/api` v1.13.0 -> v1.14.0
+  * `github.com/hashicorp/vault/sdk` v0.12.0 -> v0.13.0
+  * `github.com/ory/dockertest/v3` v3.10.0 -> v3.11.0
+  * `golang.org/x/text` v0.14.0 -> v0.18.0
+  * `github.com/hashicorp/go-secure-stdlib/plugincontainer` v0.3.0 -> v0.4.0
```

Comment:
Probably should update these with the downgrade.

```suggestion
  * `golang.org/x/text` v0.14.0 -> v0.18.0
```

Reply:
Thank you!